metadata/metadata.go

```diff
@@ -151,7 +151,11 @@ func (c *Cache) fetch(ctx context.Context, typ string, q url.Values) (_ *common.
 	}
 	defer resp.Body.Close()
 
-	var apiResp common.APIResponse
+	if resp.StatusCode != http.StatusOK {
```

Comment:
Note: This was a misc fix. I was copying some code to scrape the new web API and noticed that this endpoint does not check its status code.